app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -83,7 +85,7 @@ $govuk-cookie-banner-background-new: govuk-colour("white");
     cursor: pointer;
   }
 
-  @include govuk-media-query($from: tablet) {
+  @include govuk-media-query($from: desktop) {
     margin-top: govuk-spacing(0);
     position: absolute;
```

Comment:
I can do this at a specific size...

![Screen Shot 2019-06-12 at 13 05 46](https://user-images.githubusercontent.com/861310/59349977-dee52e00-8d12-11e9-9e81-3cda82030709.png)

You could try giving `gem-c-cookie-banner__confirmation-message` a `max-width: 90%` (or similar) on desktop?